kyt/dobi.yaml

```diff
@@ -4,7 +4,7 @@
 mount=mount-kyt-src:
   bind: "kyt/"
   path: "/home/user/src"
-  read-only: true
+  read-only: false
```

Comment:
Why has this been change to false? It should be true

Reply:
Go modules can be added during development process. As dobi is a development tool, it is required that dobi can update go.mod file.

Reply:
Ok